c2corg_ui/static/js/map/baselayerselector.js

```diff
@@ -285,4 +232,44 @@ app.BaselayerSelectorController.prototype.createIgnSource_ = function(layer) {
 };
 
 
+/**
+ * @param {string} layer Swisstopo layer name.
+ * @return {ol.source.XYZ}
+ * @private
+ */
+app.BaselayerSelectorController.prototype.createSwisstopoSource_ = function(layer) {
+  return new ol.source.XYZ({
+    attributions: [
+      new ol.Attribution({
+        html: '<a target="_blank" href="http://www.swisstopo.admin.ch/' +
+        'internet/swisstopo/en/home.html">swisstopo</a>'
+      })
+    ],
+    urls: ['10', '20'].map(function(i) {
```

Comment:
The doc says that 
> You have to use the hosts wmts{10-14}.geo.admin.ch. This is done to avoid parsing every requests to determine which are using native tiles and which are using reporjected tiles.
See https://api3.geo.admin.ch/services/sdiservices.html#other-projections